### ISSUE TITLE
node <15.0.0

### ISSUE THREAD

**Issue by n33pm** [kyeotic/raviger#79]
Hello! 👋

is there any reason why node 15 is not supported?

cheers.patrick

**Comment by kyeotic** [comment]
Odd numbered node versions are experimental. While I don't officially support experimental versions, its unlikely to have any issues.

**Comment by bsl-zcs** [comment]
Why explicitly disallow it then?  Even for those who use the library only client-side and never execute it with node.

For a project with no native-code dependencies a breakage in future node versions is not expected anyway.

**Comment by kyeotic** [comment]
@bsl-zcs I think we have different understandings of the purpose and meaning of the `engines` field. [From the docs](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#engines)

> Unless the user has set the engine-strict config flag, this field is advisory only and will only produce warnings when your package is installed as a dependency.

I am not explicitly disallowing it, I am explicitly stating the supported versions. If you want to use an experimental version of node then you are going to have to live with a non-blocking warning or two.

**Comment by bsl-zcs** [comment]
It just doesn't seems right when out of 995 packages installed in my project only raviger complains about version of node.  

Given that in this particular project raviger runs only in a browser and couldn't care less about what am I using to run webpack with.

By the way, thanks a lot for your library!  I'm recommending it over hookrouter on every occasion.

**Comment by kyeotic** [comment]
When you say "complain" do you mean the warning, or is it generating an error? If there is an actual error then I would like to address it. If it's a warning then I'm a little surprised I'm the only one; I feel like I'm interpreting the `engines` docs correctly.

Thank you for the recommendations, I truly appreciate it :)

**Comment by bsl-zcs** [comment]
Yes, it's only a warning.  And I certainly can get along with it, it's just kinda strange.

Here's complete output of `npm i` on clean install (without node_modules and package-lock):

```
>npm i
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'raviger@2.0.0',
npm WARN EBADENGINE   required: { node: '>=10.0.0 <15.0.0' },
npm WARN EBADENGINE   current: { node: 'v15.5.0', npm: '7.3.0' }
npm WARN EBADENGINE }
npm WARN deprecated axios@0.18.1: Critical security vulnerability fixed in v0.21.1. For more information, see https://github.com/axios/axios/pull/3410

added 995 packages, and audited 996 packages in 39s

80 packages are looking for funding
  run `npm fund` for details

2 high severity vulnerabilities

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.
```
 
I've tried it with both `main` and [`browser`](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#browser) fields in my `package.json` - no difference in output.   I don't know any other method to differentiate requirements for client and server sides. Can't call myself an expert in npm packaging though.

**Comment by kyeotic** [comment]
Ok, I've audited a [few](https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/package.json) [popular](https://github.com/facebook/react/blob/master/package.json) [packages](https://github.com/sindresorhus/np/blob/master/package.json#L9) and it seems using less than is... less than popular. So I will be removing it shortly

**Comment by kyeotic** [comment]
Released under [2.0.1](https://github.com/kyeotic/raviger/releases/tag/v2.0.1)
